tsinfer/formats.py

```diff
@@ -69,6 +69,16 @@ def remove_lmdb_lockfile(lmdb_file):
         os.unlink(lockfile)
 
 
+def allele_counts(genotypes):
+    """
+    Return summary counts of the number of different allele types for a genotypes array
+    """
+    n_known = np.sum(genotypes != tskit.MISSING_DATA)
```

Comment:
Returning this as a namedtuple would be nice. Client code could then write ``counts.known``, etc.

Reply:
Good idea